src/libsync/syncresult.cpp

```diff
@@ -51,19 +51,19 @@ QString SyncResult::statusString() const
         re = QLatin1String("Error");
         break;
     case SetupError:
-        re = QLatin1String("SetupError");
+        re = QLatin1String("Setup error");
         break;
     case SyncPrepare:
-        re = QLatin1String("SyncPrepare");
+        re = QLatin1String("Sync prepare");
```

Comment:
Maybe "Preparing to sync" can work better here?

Reply:
Changed it to "preparing to sync" now